lib/commands/mutemenu.js

```diff
@@ -43,6 +43,10 @@ export default {
         await logs.send(`:mute: ${buildUserDetail(user)} was muted by **${interaction.user.tag}**.`)
       }
 
+      if (hasRole(member, config.roles.verified)) {
+        await removeRole(member, guild.roles, config.roles.verified)
+      }
```

Comment:
We could just call `removeRole(member, guild.roles, config.roles.verified)` directly, since it can safely be run regardless of whether the user has the given role.

P.S. The assignment to `foundRole` within `removeRole()` can also be replaced with the new `hasRole()` method.

Reply:
I wasn't sure if the log of removeRole trying to remove a non-existant role was pertinent or not.